packages/eslint-config-base/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "@shoutem/eslint-config-base",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "description": "Shoutem's base JS ESlint config",
   "main": "index.js",
   "author": "Shoutem",
   "license": "MIT",
   "dependencies": {
     "eslint-config-airbnb-base": "14.0.0",
-    "eslint-config-prettier": "^6.10.0"
+    "eslint-config-prettier": "6.10.1"
   },
   "peerDependencies": {
-    "eslint": "^6.8.0",
-    "eslint-plugin-import": "^2.25.2",
+    "eslint": "6.8.0",
+    "eslint-plugin-import": "2.24.2",
```

Comment:
^2.25.2 resolves to 2.24.2? I assume this may be an error since I see 2.24.2 used in the other package.json

Reply:
Copy pasted this one to match other versions, will update